docs: incorporate comments from team review (#200)
Open-Attestation/documentation-website: docs/verify-section/main-properties.md

id: main-properties
title: About the verification properties
sidebar_label: Overview
custom_edit_url: null
---

To gain a comprehensive understanding of how the verification process works, look into the various properties the OA framework provides for your documents:

- [Document integrity](/docs/verify-section/document-integrity): OpenAttestation ensures that the content of the document has not been modified since the document was created, with the exception of data removed using the built-in obfuscation mechanism.
- [Issuance status](/docs/verify-section/issuance-status): OpenAttestation checks that the document has been issued and its issuance status is in good standing (for instance, it hasn't been revoked). As of today, OpenAttestation supports two ways to issue documents: DID signing and Ethereum smart contracts.

> - [Issuance identity](/docs/verify-section/issuance-identity): OpenAttestation checks and returns the identity of the issuer. By default, OpenAttestation uses DNS to verify the identity, but you have the option to use a temporary DNS record. It's important to note that OpenAttestation does not endorse any issuers. It only verifies that the issuing party in the document has provided some sort of proof that it is the same party as claimed - for example, proving ownership over a domain by creating a DNS record.

Reviewer: Let me double check with Kyle on this and get back to you

Reviewer: Ok, please change it to the following:
```suggestion
- [Issuance identity](/docs/verify-section/issuance-identity): OpenAttestation checks and returns the identity of the issuer. OpenAttestation uses DNS to verify the identity by querying the TXT records of the issuer declared within the document. It's important to note that OpenAttestation does not endorse any issuers. It only verifies that the issuing party in the document has provided some sort of proof that it is the same party as claimed - in this case proving ownership over a domain by creating DNS TXT records.
```

Author: made it bold font to emphasize the importance with minor edit
